elastic/beats
# [Metricbeat] Add dashboard for googlecloud storage metricset

## What does this PR do?

This PR is to add dashboard for googlecloud storage metricset.


## Checklist


## How to test this PR locally

1. Make sure your GCP account for testing has storage buckets
2. Enable googlecloud module: `./metricbeat modules enable googlecloud`
3. Edit modules.d/googlecloud.yml to:
```
- module: googlecloud
  metricsets:
    - storage
  project_id: "your project id"
  credentials_file_path: "your JSON credentials file path"
  exclude_labels: false
  period: 5m
```
4. Start metricbeat: `./metricbeat -e`
5. Wait for 5 minutes and you should see data points in GCP storage dashboard.

## Screenshots
<img width="2558" alt="metricbeat-googlecloud-storage-overview2" src="https://user-images.githubusercontent.com/14081635/81025538-352ff380-8e34-11ea-901f-6dd93e464ba5.png">
